iothub/service/src/FeedbackBatch.cs

```diff
@@ -11,10 +11,19 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class FeedbackBatch
     {
+        /// <summary>
+        /// A timestamp that indicates when the feedback message was received by the hub.
+        /// </summary>
         public DateTime EnqueuedTime { get; set; }
 
+        /// <summary>
+        /// A collection of feedback records across multiple devices and messages for an IoTHub.
```

Comment:
Not sure about this one, but should IoTHub be IoT hub?

Reply:
.. or IoT Hub

Reply:
I've always preferred IoT Hub, but when it comes to official MS docs, they say when talking about a hub instance to treat it like a common noun, so IoT hub.

As far as our doc comments go, we probably should go with the official MS docs standard.

Reply:
So what should we use? IoT hub ?